plugins/user-settings/config.d.ts

```diff
@@ -0,0 +1,52 @@
+/*
```

Comment:
What is the reason you added this file in this plugin? 😄

Reply:
Because if you do not have this plugin, you cannot really use feature flags. What would be your suggestion for the place for these?

Reply:
Moved to core-app-api